deploy/build-server/build-push-docker-hub.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+DOCKER_BUILDKIT=1 docker build -t stamtc/devops-build-agent:latest .
```

Comment:
We should think about actually versioning these images, so we can use explicit tags.  It would make rollback to the previous version fairly simple.

Reply:
added short commit hash as label

Reply:
will look into adding a guard for uncommitted changes to the dockerfile tomorrow